0_From_Code_First_To_First_Principles/1_code_first/assignment/src/user/userController.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response, NextFunction } from "express";
+import { UserService } from "./userService";
+
+export class UserController {
+	static async createUser(req: Request, res: Response, next: NextFunction) {
+		const userService = new UserService();
+
+		try {
+			const user = await userService.createUser(req.body);
+			res.json(user);
```

Comment:
Just noting that it doesn't seem like you are:
1. including a status code in your response
2. returning the response in the requested format (e.g. `{ error: ''UserNotFound', data: undefined, success: false }`)

Reply:
Are you referring to error responses? I'm handling all the errors using middleware in the `app.ts` file